src/test/java/com/google/javascript/clutz/partial/inner_typedef_npe.d.ts

```diff
@@ -0,0 +1,15 @@
+declare namespace ಠ_ಠ.clutz {
+  class module$exports$odd$npe$typedef extends module$exports$odd$npe$typedef_Instance {
+  }
+  class module$exports$odd$npe$typedef_Instance {
+    private noStructuralTyping_: any;
+    foo ( ) : ಠ_ಠ.clutz.some.type < ಠ_ಠ.clutz.module$contents$odd$npe$typedef_ATypeDef > ;
+  }
+}
+declare module 'goog:odd.npe.typedef' {
+  import alias = ಠ_ಠ.clutz.module$exports$odd$npe$typedef;
+  export default alias;
+}
+declare namespace ಠ_ಠ.clutz {
+  type module$contents$odd$npe$typedef_ATypeDef = { a : string } ;
```

Comment:
What would we have seen with the old emit here?

Reply:
This block would have been missing. Clutz would have crashed with NPE because emitMissingTypes would be working with a TypedVar object that has `.getType()` returning `null` as it does for `typedefs`.